lib/fetch_board.rb

```diff
@@ -1,4 +1,6 @@
-class DisplayBoard
+# frozen_string_literal: true
+
+class FetchBoard
   attr_reader :state
 
   def initialize(board_gateway)
```

Comment:
For line 8, could change it to `@state = [nil] * 9` so you it's easier to change in the future if need be and so a reader can instantly know that there are 9 values (instead of counting to figure it out). Not really sure if that's more readable though...

Reply:
I think that is a lot more readable, thank you.